### PR TITLE
Remove completed

### PR DESCRIPTION
**Pull request by whid-bb**
- Added remove completed tasks functionality

[Live Demo](https://whid-bb.github.io/to-do-list/)

### CODE REVIEW

File: src/js/Complete.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  RemoveItems = new RemoveItems();
+
+  clearCompleted = () => {
+    const completed = document.querySelectorAll(`.${globals.TASK_LIST_COMPLETED}`);
+    completed.forEach((el) => {
+      this.RemoveItems.removeItem(el);
+    });
+  };
+
+  taskComplete = (parent, id) => {
```
Comment (Ismailco) [inline src/js/Complete.js:20]: - [ ] The task ``` completed``` should be a boolean, ```true``` or ```false``` not 1 or 0, please fix this

Screenshot_20220603_055725

File: src/js/Complete.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import Ls from './Ls.js';
+import * as globals from './globals.js';
+import RemoveItems from './Remove.js';
+
+class Complete {
+  Ls = new Ls();
+
+  RemoveItems = new RemoveItems();
+
+  clearCompleted = () => {
```
Comment (Ismailco) [inline src/js/Complete.js:10]: - [ ] Good work implementing this function, however, you should use ``` Array.filter()``` method to implement the function for the "Clear all completed" button. Kindly make these changes to your code so it meets the requirements

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @whid-bb,

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/js/ListItem.js [hunk 3]
```diff
@@ -35,26 +16,33 @@ class ListItem extends Ls {
...
     const trashIco = e.currentTarget.querySelector('.trash-ico');
+    this.list = this.getFromLS();
 
     trashIco.currentParentEl = e.currentTarget;
     trashIco.currentParentId = numId;
-    trashIco.addEventListener('click', this.removeItem);
-
+    trashIco.addEventListener('click', (e) => {
+      this.RemoveItems.removeItem(e.target);
+      // this.RemoveItems.recalculateIndexes();
```
Comment (Mihndim2020) [inline src/js/ListItem.js]: - [OPTIONAL] Please kindly remove `commented code` from your `code base`, as it makes your `code base` dirty and less readable. To make sure that your codebase is always clean, always remember to remove commented-codes.

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @whid-bb, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**